### DISCUSSION TITLE
Recommended ROS version

### DISCUSSION THREAD

**Discussion by kjrusscher** [iRobotEducation/create3_docs#444] [Questions] [answered]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

A.0.0

### Which RMW is your robot running?

CycloneDDS

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

The overview of the [firmware releases](https://iroboteducation.github.io/create3_docs/releases/overview/) says: 
`As of March 2023, use of Galactic is recommended, as it is more performant than Humble on Create 3.`

Wat is the current status? Is Humble already recommended? If not, when can we expect it to be recommended?

**Comment by bperseghetti** [comment, marked as answer]
Well, considering that [Galactic is EOL as of Dec 09 2022 ](https://docs.ros.org/en/iron/Releases/End-of-Life.html) I would say Humble is/should be the only recommended one to be using!

Also depending on your "compute platform" the upgrade to the RMW (and associated matching DDS vendors) in combination with the other continual Humble updates from the last year make it actually perform much better than galactic in every category I can think of.

So to be more concise "Humble is ready and recommended"

**Reply by shamlian** [reply]
Just because a ROS version is EOL doesn't mean it's unused by the community. It is true that Galactic had been more performant than Humble, in general. That said, except for the increased startup time (which can be seen in the details if you drill into a particular Humble release), for ROS 2 users interested in using Humble, there is no reason not to use the most recent Humble firmware release. Beginner users, especially ones who are interested in using the robot only for the PWP, may be confused or put off by this increased start time, and so we had recommended to use Galactic.

To your point, it's a goal of the team to continue to improve the performance of Humble on the robot; H.2.2 dramatically improved the runtime performance, and perhaps that could be reason enough to remove the warning. This, coupled with a future update solving the startup condition, would likely result in the removal of this warning.

**Comment by kjrusscher** [comment]
Thanks for your answer. I would be nice if they would update this section.

**Reply by shamlian** [reply]
Your PR is welcome. :)
